inst/stan/functions/spatial_functions.stan

```diff
@@ -0,0 +1,12 @@
+/**
+  * File to call all desired spatial functions for use with expose_functions()
```

Comment:
Did you use this in `generate_simulated_data.R` instead of calling `wwinference.stan`? If so, I must have missed it

Reply:
I used it to reduce the functions used in the function block of wwinference.stan.  This is regarding Dylans last request from above.

Reply:
But so you'll want to replace:
```
  model <- cmdstanr::cmdstan_model(
    stan_file = system.file(
      "stan", "wwinference.stan",
      package = "wwinference"
    ),
    compile = TRUE,
    compile_standalone = TRUE,
    force_recompile = TRUE
  )
  model$expose_functions(global = TRUE)
```

With 
  model <- cmdstanr::cmdstan_model(
    stan_file = file.path("inst", "stan","functions", "spatial_functions.stan"),
    compile = TRUE,
    compile_standalone = TRUE,
    force_recompile = TRUE
  )
  model$expose_functions(global = TRUE)